### PR TITLE
Upadate russian locale

### PR DESCRIPTION
**Pull request by kussakaa**
## Status

(pick one)

- READY: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description
A small correction to the Russian translation

### CODE REVIEW

File: Locales/Russian_ru.json [hunk 1]
```diff
@@ -13,1301 +13,1316 @@
...
+  "Surrender":"Сдаться",
+  "Surrender to the enemy?":"Сдаться противнику?",
+  "Voting for surrender\nis not allowed before the game starts.\n":"Сдаваться до начала игры запрещено.",
+  "Voting for surrender\nrequires a {NEXTMAP_MINS} min wait\nafter each started vote\nto prevent spamming.\n":"После каждого голосования\nнужно ждать {NEXTMAP_MINS} мин,\nчтобы избежать спама.",
+  "Voting for surrender\nis not available as a spectator\n":"Будучи зрителем, нельзя сдаваться.",
+  "Vote Surrender":"Сдаться",
+  "Vote to end the game\nin favour of the enemy team.\n\n- report any abuse of this feature.\n\nTo Use:\n\n- select surrender if you're sure.\n- everyone votes.\n":"Голосование за окончание игры\nв пользу вражеской команды.\n\n- сообщайте о любом злоупотреблении этой функцией.\n\nКак использовать:\n\n- сдавайтесь только если вы уверены.\n- голосуют все.\n",
+  "We Surrender! (I'm sure)":"Мы сдаёмся! (Я уверен)",
+  //vote scramble
+  "Scramble":"Схватка",
```
Comment (AsuMagic) [inline Locales/Russian_ru.json:1170]: English translations I can find to this word seems to hint towards the meaning "fight"/"skirmish". Is this the actually correct translation for "scramble", which in KAG means "randomly shuffling teams"?

File: Locales/Russian_ru.json [hunk 2]
```diff
@@ -13,1301 +13,1316 @@
...
 
 {
+
 //// SCOREBOARD ////
 //ScoreboardCommon.as
-	"Match time: {TIME}": "Время матча: {TIME}",
-	"h ": "ч ",
-	"m ": "м ",
-	"s ": "с ",
+  "Match time: {TIME}": "Время матча: {TIME}",
```
Comment (AsuMagic) [inline Locales/Russian_ru.json:19]: Tabs should not have been modified to spaces in this PR, I will fix that.

**Review by AsuMagic — CHANGES_REQUESTED** [review]

File: Locales/Russian_ru.json (hunk 2 above)
Comment (kusakich) [reply Locales/Russian_ru.json:19]: Ok, sorry

File: Locales/Russian_ru.json (hunk 1 above)
Comment (kusakich) [reply Locales/Russian_ru.json:1170]: Yes, "схватка" thisis fight, skirmish

**Review by AsuMagic — APPROVED** [review]

**Pull request merged**